packages/mobile/src/screens/sign-on-screen/components/AccountHeader.tsx

```diff
@@ -0,0 +1,182 @@
+import { css } from '@emotion/native'
+import { BlurView } from '@react-native-community/blur'
+import type { ImageURISource } from 'react-native'
+import { ImageBackground, View } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import {
+  Avatar,
+  Flex,
+  IconCamera,
+  IconImage,
+  IconVerified,
+  Text,
+  useTheme
+} from '@audius/harmony-native'
+
+type AccountHeaderProps = {
+  onSelectCoverPhoto: () => void
+  onSelectProfilePicture: () => void
```

Comment:
These should be nullable for later screens
```suggestion
  onSelectCoverPhoto?: () => void
  onSelectProfilePicture?: () => void
```